Dockerfile

```diff
@@ -22,6 +25,12 @@ RUN pip install -r /zap/requirements.txt \
 	&& chown -R zap:zap /zap/ \
 	&& chmod +x /zap/zap-baseline-custom.py
 
+RUN chgrp -R 0 /zap

```

Comment:
I think we should let the base image handle these lines 28-32: https://github.com/zaproxy/zaproxy/blob/main/docker/Dockerfile-weekly

Reply:
ok